File: Scala/src/main/scala/TriviaQuestion.scala
import java.util

import com.fasterxml.jackson.annotation.JsonProperty

class TriviaQuestion {
  @JsonProperty("question") val question: String = null
  @JsonProperty("correct_answer") val correctAnswer: String = null
  @JsonProperty("incorrect_answers") val incorrectAnswers: util.List[String] = null
  @JsonProperty("category") private val category: String = null
  @JsonProperty("type") private val `type`: String = null
  @JsonProperty("difficulty") private val difficulty: String = null
}

File: Groovy/src/main/groovy/com/codevalue/archnext/TriviaQuestion.java
package com.codevalue.archnext;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class TriviaQuestion {
    @JsonProperty("question")
    private String question;

    @JsonProperty("correct_answer")
    private String correctAnswer;

    @JsonProperty("incorrect_answers")
    private List<String> incorrectAnswers;

    @JsonProperty("category")
    private String category;

    @JsonProperty("type")
    private String type;

    @JsonProperty("difficulty")
    private String difficulty;

    public String getQuestion() {
        return question;
    }

    public String getCorrectAnswer() {
        return correctAnswer;
    }

    public List<String> getIncorrectAnswers() {
        return incorrectAnswers;
    }
}
